bpf/lib/ipv4.h

```diff
@@ -8,6 +8,30 @@
 
 #include "dbg.h"
 
+struct ipv4_frag_id {
+	__be32	daddr;
+	__be32	saddr;
+	__be16	id;		/* L4 datagram identifier */
+	__u8	proto;
+	__u8	pad;
+} __attribute__((packed));
```

Comment:
nit: we should have __packed available as macro.

Reply:
We do, although it is not used in the code at the moment. And it didn't exist when I pushed this code to the PR :astonished:. Sure, will change!

Reply:
Oh, I wasn't aware, no worries then. :-) Probably another one for bpf/ wide cleanup to convert all `__attribute__((packed))` into `__packed`. If you prefer I could also do the packed and define one later for all bpf/, up to you.

Reply:
I'll do the cleanup.